package-dev/Runtime/SentryInitialization.cs

```diff
@@ -107,19 +107,45 @@ public string Platform
         public Il2CppMethods Il2CppMethods => _il2CppMethods;
 
         private Il2CppMethods _il2CppMethods
-// Lowest supported version to have all required methods below
+            // Lowest supported version to have all required methods below
 #if !ENABLE_IL2CPP || !UNITY_2020_3_OR_NEWER || !UNITY_64
             ;
 #else
             = new Il2CppMethods(
                 il2cpp_gchandle_get_target,
-#if UNITY_2021_3_OR_NEWER
-                il2cpp_native_stack_trace,
-#else
                 Il2CppNativeStackTraceShim,
-#endif
                 il2cpp_free);
 
+        private static string SanitizeDebugId(string debugId)
+        {
+#if UNITY_ANDROID || UNITY_STANDALONE_LINUX
+            // For ELF platforms, the 20-byte `NT_GNU_BUILD_ID` needs to be
+            // turned into a "little-endian GUID", which means the first three
+            // components need to be byte-swapped appropriately.
+            // See: https://getsentry.github.io/symbolicator/advanced/symbol-server-compatibility/#identifiers
+
+            // As our id is already stringified, we will use substrings here,
```

Comment:
It's strigified already below:

```
            imageUUID = new String(uuidBuffer, 0, Array.IndexOf(uuidBuffer, '\0'));
```

Is it possible to do this without the 18+ heap allocations?